### ISSUE TITLE
Does Fornax support incremental builds?

### ISSUE THREAD

**Issue by panesofglass** [ionide/Fornax#61]
**Is your feature request related to a problem? Please describe.**
I would like to host Fornax in an Azure Function and have it re-generate new content when requested, but I would want this to be fast and only generate the new content.

**Describe the solution you'd like**
Support incremental content generation.

**Describe alternatives you've considered**
Accept what exists or use Gatsby, which does support incremental builds.

**Additional context**
I assumed this might already exist for watch but didn’t see it listed.

**Comment by Krzysztof-Cieslak** [comment]
No, Fornax doesn't support incremental builds - it always regenerates whole content. TBF, I wouldn't recommend hosting long-running Fornax process this way - it's not designed nor tested to work like that.
